woocommerce/woocommerce-android
# Add automated beta uploads

**To Test**
1) Checkout this branch.
2) Update the `versionName` and `versionCode` to `3.5-rc-2` and `106`. Commit your changes.
3) Run `bundle exec fastlane build_pre_releases`.
4) Run `bundle exec fastlane upload_beta_build_to_play_store`.
5) Open the Google Play store – see that there is now a new release that hasn't been rolled out, and that it has your new `versionCode`. The script doesn't currently handle release notes, so this should remain empty.
6) Discard the release at the bottom of the screen, then go to the artifact library and delete the binary.

**Update release notes:**
